pkg/verifier/aws/entry_point.go

```diff
@@ -18,6 +19,9 @@ import (
 	"github.com/openshift/osd-network-verifier/pkg/verifier"
 )
 
+//go:embedd userdata-template.yaml
```

Comment:
Typo here in `embed`
```suggestion
//go:embed userdata-template.yaml
```